src/NServiceBus.StructureMap.Tests/NServiceBus.StructureMap.Tests.csproj

```diff
@@ -1,23 +1,28 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net452</TargetFramework>
+    <TargetFrameworks>net452;netcoreapp2.0</TargetFrameworks>
     <DebugType>full</DebugType>
     <DebugSymbols>True</DebugSymbols>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
   <ItemGroup>
     <Compile Remove="NServiceBus.ObjectBuilder.StructureMap.approved.cs" />
-    <PackageReference Include="ApprovalTests" Version="3.0.13" />
-    <PackageReference Include="ApprovalUtilities" Version="3.0.13" />
     <PackageReference Include="StructureMap" Version="4.5.1" />
-    <PackageReference Include="Mono.Cecil" Version="0.10.0-beta6" />
-    <PackageReference Include="NServiceBus" Version="7.0.0-alpharelease0051" />
+    <PackageReference Include="Mono.Cecil" Version="0.10.0-*" />
+    <PackageReference Include="NServiceBus" Version="7.0.0-*" />
     <PackageReference Include="NUnit" Version="3.7.1" />
-    <PackageReference Include="ApiApprover" Version="6.1.0-beta2" />
-    <PackageReference Include="PublicApiGenerator" Version="6.1.0-beta2" />
-    <PackageReference Include="NServiceBus.ContainerTests.Sources" Version="7.0.0-alpharelease0051" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.7.0" />
+    <PackageReference Include="NServiceBus.ContainerTests.Sources" Version="7.0.0-*" />
     <ProjectReference Include="..\NServiceBus.StructureMap\NServiceBus.StructureMap.csproj" />
+    <Compile Remove="APIApprovals.cs" />
+    <Compile Remove="ApprovalTestConfig.cs" />
```

Comment:
these two lines should be in a `<ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.0'">`